app/forms/providers/link_application/confirm_link_form.rb

```diff
@@ -1,20 +1,15 @@
 module Providers
   module LinkApplication
     class ConfirmLinkForm < BaseForm
-      form_for LegalAidApplication
+      form_for LinkedApplication
 
-      attr_accessor :link_case
+      attr_accessor :confirm_link
 
-      validates :link_case, presence: true, unless: :draft?
+      validates :confirm_link, presence: true, unless: :draft?
```

Comment:
Are we still using the link_case column on the legal_aid_applications table for anything? If not should it be removed?

Reply:
I was thinking of re-using it for once the change is confirmed we set it true or false for reporting purposes, while deleting the linked_application object if it's false :+1: